modules/installation-aws-about-government-region.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-government-region.adoc
+
+[id="installation-aws-about-government-region_{context}"]
+= AWS Government and air-gapped regions
+
+{product-title} supports deploying a cluster to AWS Government and other
+air-gapped regions. These hidden regions do not have published {op-system-first}
+AMIs to select, so you must upload a custom AMI that belongs to that region.
+
+Common AWS regions that do not have published {op-system} AMIs include:
+
+* AWS GovCloud, Secret, Top Secret and other secret cloud services
+* Specialized AWS public commercial services
+* Other hidden regions
+
+The following AWS GovCloud partitions are supported:
+
+* `us-gov-west-1`
+* `us-gov-east-1`
+
+When deploying to a region without a published {op-system} AMI, you must specify
+the custom endpoints that are required to access required AWS APIs.
```

Comment:
Custom endpoints are required when the region is hidden. AWS Gov cloud regions are not hidden therefore do not require specifying service endpoints.

Secondly providing AMI is not related to service endpoints. like AWS Govcloud where users need to specify AMI but don't need to specify service endpoints.